lang_java/parsing/tree-sitter-parser

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
I would just add a line or two of comment here saying what this is used for!

Reply:
Ok, the documentation's all in CONTRIBUTING.md, I didn't know if we wanted the bash scripts to be cluttered with comments as well.